prometheus-webconsole/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.secondbase</groupId>
+        <artifactId>secondbase</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>prometheus-webconsole</artifactId>
+    <packaging>jar</packaging>
+    <name>prometheus webconsole</name>
+    <dependencies>
+        <dependency>
+            <groupId>org.secondbase</groupId>
+            <artifactId>webconsole</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.secondbase</groupId>
+            <artifactId>flags</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.secondbase</groupId>
+            <artifactId>core</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient</artifactId>
```

Comment:
I would suggest we put the prometheus simpleclient in the core module along with the version, so that we can keep metrics from our submodules. Like request latency for the webconsole requests and log counters, etc.